### ISSUE TITLE
Implement collapsing section for the delegate crash course

### ISSUE THREAD

**Issue by SAuroux** [thewca/worldcubeassociation.org#637]
The delegate crash course grows more and more and meanwhile, the overview on the content suffers. I would love to have collapsing sections for the document (see http://getbootstrap.com/javascript/#collapse-example-accordion), to maintain a decent overview.

**Comment by jfly** [comment]
I can see us implementing this as a bit of javascript that searches for `<h2>` tags, or as a custom markdown plugin. Unfortunately, I think either way is going to be a bit tricky. If someone wants to take this on, please reach out to me before starting on this, because I have some thoughts about what we should do here.

**Comment by SAuroux** [comment]
It would be nice to have something that does not necessarily depend on a tag/section depth. For example, the results part has four bigger subsections right now. Personally, I would format this as four collapsing subsetions within the collapsing Results section.

**Comment by pedrosino** [comment]
@SAuroux I discussed with Jeremy, and having something like that _inside_ a markdown editable field sounds complicated.
One easy solution would be to make the crash-course not a post-like page anymore, but hard-coded instead. That way we have full control over the html and can do the accordion thing easily.
Is that too bad of an idea?

**Comment by Luis-J-Ianez** [comment]
Why not simply have the actual page as a content index linking to new sub-pages?  All of them could still be markdown.

**Comment by dunkOnIT** [comment]
We no longer have the delegate crash course on the website, and it will be superceded by an LMS soon in any case
